base_export_manager/views/ir_exports.xml

```diff
@@ -29,52 +29,48 @@
         <field name="model">ir.exports</field>
         <field name="arch" type="xml">
             <form string="Export Profile">
-                <sheet>
```

Comment:
You're dismissing the changes of the previous commit here.

Reply:
Thanks for the advice, now it should be correct